lochness/redcap/__init__.py

```diff
@@ -99,19 +107,17 @@ def initialize_metadata(Lochness: 'Lochness object',
                                  record_query,
                                  f'initializing data {study_name}')
 
-    # load pulled information as list of dictionary : data
+    # load pulled information as a list of dictionaries
     with tf.NamedTemporaryFile(suffix='tmp.json') as tmpfilename:
         lochness.atomic_write(tmpfilename.name, content)
         with open(tmpfilename.name, 'r') as f:
             data = json.load(f)
 
     df = pd.DataFrame()
-    # for each record in pulled information, extract subject ID and source IDs
+    # extract subject ID and source IDs for each sources
     for item in data:
-        if multistudy:
+        if multistudy:  # filter out data from other sites
             site_two_letters_redcap_id = item[redcap_id_colname][:2]
-            site_two_letters_study = study_name.split('_')[1]
```

Comment:
I see now that this is where we have removed the existence of `_` in study name.